app/helpers/layout_helper.rb

```diff
@@ -19,6 +19,12 @@ def show_amount_with_tooltip(amount)
     end
   end
 
+  def show_tooltip_with_text(text)
```

Comment:
@yunixon please refactor method `show_amount_with_tooltip`, so there will be no duplicated code.